### DISCUSSION TITLE
Is it possible to set specific settings (like colors, fonts, title etc.) when running a specific app?

### DISCUSSION THREAD

**Discussion by MrDowntempo** [wezterm/wezterm#5085] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

I'm on Nobara, which is based on Fedora 39, with the lasted KDE5 plasma with Wayland on nVidia drivers.

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

One of the things I love about WezTerm over even other GPU terminals, is the amount of customization you get with your fonts. I can specify a different font for italics, and then when I'm editing files in a terminal editor that supports italic comments, I get to see them in a cursive font.

But I wanna see if I can take this further. I like to play some old text based games, and when I do, I want to play them in the font I remember playing them in (not the modern sexy terminal fonts I'm otherwise using). I've done my best to find any thing helpful in the docs, but its either not there, or I'm missing it. Perhaps this isn't (yet) possible?

**Comment by beeb** [comment]
I would too find this useful to disable ligatures in my modal editor but enable them for other processes.

**Comment by nekowinston** [comment, marked as answer]
I posted a snippet showing how to change padding depending on the current fg process here, this also works with other settings: https://github.com/wez/wezterm/discussions/3839#discussioncomment-6123338

**Reply by MrDowntempo** [reply]
Thanks! I was able to adapt this code to my purposes. So cool!
